modules/administration/pages/repo-metadata.adoc

```diff
@@ -21,13 +21,13 @@ gpg --gen-key
     You can check by listing the keys in your keyring:
 +
 ----
-gpg --list-keys
+mgrctl exec -- gpg --list-keys
 ----
 +
 . Add the password for your keyring to the [filename]``/etc/rhn/signing.conf`` configuration file, by opening the file in your text editor and adding this line:
 +
 ----
-GPGPASS="password"
+mgrctl exec -- GPGPASS="password"
```

Comment:
This is not a command.  Maybe, we'd better recommend here and for the whole procedure to start a terminal with mgrterm and the run all the commands there.